src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -323,6 +334,10 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
                         {
                             return await this.HandleRewindInstanceRequestAsync(request, instanceId);
                         }
+                        else if (string.Equals(operation, RestartOperation, StringComparison.OrdinalIgnoreCase))

```

Comment:
I don't love the fact that we accept GET requests for this operation. I realize we do for Terminate/Rewind, but we should arguably fix that as a bug, as [we document](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-http-api#request-6) that it requires a `POST`.

Reply:
I see. Should I change these all to post in this PR while I'm here?

Reply:
If you have time to, I would say let's do it. Otherwise, feel free to punt to next release, but then we should file an issue.